### DISCUSSION TITLE
After closing the sponsor account and re applying, my sponsorship link will jump to page 404.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#9095] [Other Feature Feedback, Questions, & Ideas]
After closing the sponsor account, click my sponsorship link 【https://github.com/sponsors/name/dashboard】to jump to page 404
![image](https://user-images.githubusercontent.com/74201947/146683874-f3196c9d-e958-48c8-97d9-a3ece34b2cdf.png)

**Comment by denandreychuk** [comment]
same here. did anyone find out the solution?

**Reply by ghost** [reply]
I ran into the same issue. I contacted Github support and the issue was rectified within 24 hours.

**Comment by iricartb** [comment]
I have the same issue
